### PR TITLE
docs(security/helmet): add how to use helmet with fastify

### PR DESCRIPTION
**Pull request by saulotoledo**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] Other... Please describe: docs
```

## What is the current behavior?

The instructions in the documentation are insufficient if you are using `Fastify`.

Issue Number: https://github.com/nestjs/nest/issues/5195

## What is the new behavior?

The instructions are now complete.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: content/techniques/security.md [hunk 1]
```diff
@@ -20,7 +20,23 @@ import * as helmet from 'helmet';
...
+$ npm i --save fastify-helmet
+```
+
+Once the installation is complete, you'll need to register it:
+
+```typescript
+import * as helmet from 'fastify-helmet';
+// somewhere in your initialization file
+app.register(helmet);
+// or, but not type safe
```
Comment (johnbiundo) [inline content/techniques/security.md:35]:
```suggestion
// or the following, but note that it's not type safe
```

File: content/techniques/security.md [hunk 2]
```diff
@@ -20,7 +20,23 @@ import * as helmet from 'helmet';
...
+app.getHttpAdapter().getInstance().register(helmet);
```
Comment (johnbiundo) [inline content/techniques/security.md:36]:
```suggestion
// app.getHttpAdapter().getInstance().register(helmet);
```

I'm thinking that, even though you say "or the following", perhaps we want to ensure people aren't confused and use *both*.  Hence the suggestion to maybe comment this alternative form.  Thoughts?

Comment (saulotoledo) [reply content/techniques/security.md]: I agree with your suggestion. And having it as a comment is enough.

File: content/techniques/security.md (hunk 1 above)
Comment (saulotoledo) [reply content/techniques/security.md]: Changed.

**Comment by saulotoledo** [comment]
Hi @johnbiundo. Thanks for your review!
I agree with your suggestions and I changed them all. Could you please check another minor change [in this line](https://github.com/nestjs/docs.nestjs.com/pull/1401/files#diff-f1442c2121969c777e646ae42f360ed5R29)? I think it is interesting to make clear the difference between middlewares and plugins in Fastify, or the users may be confused why they are doing things in a different way.

**Comment by kamilmysliwiec** [comment]
Thanks!

**Pull request merged**